src/ci_nanovg.cpp

```diff
@@ -344,7 +344,15 @@ void Context::stroke() {
 // Text //
 
 int Context::createFont(const std::string &name, const fs::path &filepath) {
+#if defined( CINDER_UWP ) || ( defined( _MSC_VER ) && ( _MSC_VER >= 1900 ) )
+	std::wstring ws(filepath);
+  file = std::string(ws.begin(), ws.end());
```

Comment:
Curious. Why is the roundtrip through std::wstring and then std::string needed on windows? If it is, this should be submitted as a separate PR.

Reply:
I made it, becouse on windows for filesystem work, cinder use not boost library as on linux an mac but they own library.
so fs::path 

#if defined( CINDER_UWP ) || ( defined( _MSC_VER ) && ( _MSC_VER >= 1900 ) )
	namespace fs = std::tr2::sys;
} // namespace cinder
#else
	namespace fs = boost::filesystem;
} // namespace cinder